src/app/services/auth.service.ts

```diff
@@ -0,0 +1,29 @@
+import { HttpClient } from '@angular/common/http';
+import { Injectable } from '@angular/core';
+import { Observable } from 'rxjs';
+import { JwtDto } from '../models/jwt-dto';
+import { LoginUser } from '../models/login-user';
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/ispp-yourney/angular-app/pullRequest?prid=7132249)
```suggestion
import { LoginUser } from "../models/login-user";
```